backend/de.metas.fresh/de.metas.fresh.base/src/main/jasperreports/de/metas/reports/openitems_fibu/report_details.jrxml

```diff
@@ -43,14 +42,11 @@ FROM
 		i.C_BPartner_ID,
 		i.IsSOTrx,
 		i.DateInvoiced,
-		EXISTS (SELECT 0 FROM C_PaySelectionLine psl WHERE psl.processed = 'Y' AND psl.C_Invoice_ID = i.C_Invoice_ID AND psl.isActive = 'Y') AS PassForPayment,
+		EXISTS (SELECT 0 FROM C_PaySelectionLine psl WHERE psl.C_Invoice_ID = i.C_Invoice_ID AND psl.isActive = 'Y') AS PassForPayment,
```

Comment:
why removing processed=Y?

Reply:
as discussed, evaluate if we shall check C_PaySelection.DocStatus in CO, CL ... or something?